examples/src/main/java/alluxio/cli/S3ACompatibilityTest.java

```diff
@@ -0,0 +1,220 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.cli;
+
+import alluxio.underfs.s3a.S3AStreamingUploadContractTest;
+import alluxio.underfs.s3a.S3AUnderFileSystemContractTest;
+
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.DefaultParser;
+import org.apache.commons.cli.Options;
+import org.apache.commons.cli.ParseException;
+import org.junit.runner.JUnitCore;
+import org.junit.runner.Result;
+import org.junit.runner.notification.Failure;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Class to perform S3A compatibility tests.
+ */
+public final class S3ACompatibilityTest {
+  /**
+   * The main class to run the S3A compatibility tests.
+   *
+   * @param args the arguments parsed by commandline
+   */
+  public static void main(String[] args) throws Exception {
+    Options options = new Options().addOption("help", false, "Show help for this test");
+    CommandLine cmd;
+    try {
+      cmd = new DefaultParser().parse(options, args);
+    } catch (ParseException e) {
+      System.out.println("Failed to parse input args " + e);
+      return;
+    }
+    if (cmd.hasOption("help")) {
+      printHelp();
+      return;
+    }
+    runJUnitClass(S3AStreamingUploadContractTest.class);
+    runJUnitClass(S3AUnderFileSystemContractTest.class);
+  }
+
+  /**
+   * Prints the help message.
+   */
+  private static void printHelp() {
+    System.out.println("bin/alluxio runS3Tests");
+    System.out.println("Test if the target storage system which declares it's S3 compatible "
+        + "can fulfill the minimum S3A compatibility requirements in order to work with Alluxio "
+        + "through Alluxio's integration with S3. A test bucket `testS3ABucket` is required to "
```

Comment:
nit: remove "through Alluxio's integration with S3"
nit: "A test bucket `testS3ABucket`" -> "A test bucket specified by the option `testS3ABucket`"